File: BTSuggestions/BTSuggestions.Core/Entities/ContentEntity.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace BTSuggestions.Core.Entities
{
    public class ContentEntity : BaseEntity
    {
        [Required]
        public string Content { get; set; } 
        [JsonIgnore]
        public virtual UserEntity User { get; set; }
        [Required]
        [ForeignKey("UserId")]
        public int UserId { get; set; } //ForiegnKey to UserId
    }
}

File: BTSuggestions/BTSuggestions.Web/ClientApp/src/entity/ContentEntity.ts
import UserEntity from "./UserEntity";
export default class ContentEntity {
    contentId: number;
    content: string;
    user: UserEntity;

    constructor(JSONData: any) {
        this.contentId = JSONData.contentId;
        this.content = JSONData.content;
        this.user = JSONData.user;
    }
}